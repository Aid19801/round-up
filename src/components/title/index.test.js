// import React from 'react';
// import Title from './index';
// import Link from '../Link.react';
// import renderer from 'react-test-renderer';

// it('renders correctly', () => {
//   const tree = renderer
//     .create(<Title text="mock-text" />)
//     .toJSON();
//   expect(tree).toMatchSnapshot();
// })