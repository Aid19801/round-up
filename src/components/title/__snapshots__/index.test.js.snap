// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage matches the snapshot 1`] = `
<div
  className="title"
>
  <h4>
    mock-text
  </h4>
</div>
`;

exports[`Title matches the snapshot 1`] = `
<div
  className="title"
>
  <h4>
    mock-text
  </h4>
</div>
`;
